### DISCUSSION TITLE
How to request to register this community

### DISCUSSION THREAD

**Discussion by churros0730** [EddieHubCommunity/support#9025] [Q&A] [answered]
As a frontend developer, I'd like to join in this community and contribute my skills to improve the app.
Please add me a new member. Thanks.

**Comment by khanovico** [comment]
As a 7 years experienced AI engineer, I want to join in this community and dedicate to the latest projects and handle unknown issues. Please accept me as a new member. Thanks.

**Comment by Pradumnasaraf** [comment, marked as answer]
Hey @churros0730 @churros0730,

You don't need any kind of registration. You can join and get involved in the community by joining the GitHub organizations as well as the Discord. You will find instructions here: https://github.com/EddieHubCommunity?view_as=public
